odl/tomo/operators/ray_trafo.py

```diff
@@ -340,8 +354,8 @@ def __init__(self, domain, geometry, **kwargs):
             - ``'skimage'``: scikit-image, only 2D parallel with square
               reconstruction space.
 
-            For the default ``None``, the fastest available back-end is
-            used.
+            For the default ``None``, the backend is chosen in the order listed
+            above, e.g. the fastest available back-end is used.
```

Comment:
e.g.?
I'd use the old formulation and add ", tried in the above order."

Reply:
That also works!